### PR TITLE
Update to Bulma v0.8.2

### LINKED ISSUE #237: Update to Bulma v0.8.2
This is an issue to track features introduced in Bulma 0.8.0.  

- [x] Support light/dark variants of main colors
- [x] Support panel colors
- [x] New `.is-fullwidth` modifier for `image`

### PR DESCRIPTION
**Pull request by MrCreeper1008**
Closes #237  

Changes:
- `<Panel />` component now has a `color` prop that accepts the same value as other components that support color like `<Button />`
- New `colorVariant` modifier that accepts two values: `light` and `dark`. Equivalent to Bulma's `is-light` and `is-dark` modifiers.
- Added a changelog.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-bulma-components",
-  "version": "3.2.0",
```
[the remark this answers is not in the document]
Comment (MrCreeper1008) [reply package.json:3]: oh didn't realize this. will fix. thank you!

**Comment by ndrez** [comment]
Hey Guys, 
This is a pretty need update, I was considering to do it my self. 
How is this going? 
Do you need some help with this maybe?  
Best,
Endre

**Comment by MrCreeper1008** [comment]
@ndrez This PR is almost finished. I am just waiting on the review. We do need help with TypeScript definition though (#207). If you have any experience with TypeScript, we would really appreciate your help!

**Comment by couds** [comment]
@MrCreeper1008 Good work! thanks for this update =)

**Comment by ndrez** [comment]
@couds thanks for the quick approval. 
@MrCreeper1008 don't have a lot experience with ts. If you get stuck with something, I may however have someone to support you.

**Comment by MrCreeper1008** [comment]
@ndrez we also need help with writing the documentation, so you can also help with that. I haven't started writing it yet but one thing I'm sure is that it would be very tedious so any help would be greatly appreciated. I really appreciate your support!

**Pull request merged**